pkg/service/service.go

```diff
@@ -28,6 +28,29 @@ import (
 	pipelinePB "github.com/instill-ai/protogen-go/vdp/pipeline/v1alpha"
 )
 
+type TextToImageInput struct {
+	Prompt   string
+	Steps    int64
+	CfgScale float32
+	Seed     int64
+	Samples  int64
+}
+
+type TextGenerationInput struct {
+	Prompt        string
+	OutputLen     int64
+	BadWordsList  string
+	StopWordsList string
+	TopK          int64
+	Seed          int64
+}
+
+type VisionInput struct {
```

Comment:
```suggestion
type ImageInput struct {
```

I am aware that the `model-backend` is using `VisionInput` which I think should be changed to `ImageInput` to be more precise.